### DISCUSSION TITLE
Archimede organization halted in the making

### DISCUSSION THREAD

**Discussion by alebas64** [community/community#113901] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

## Thesis

Me and my friend @ripu would like to create a organizzation for a big upcoming project and we want to use the name "archimede" for it. Unfortunatly it seams there is a dormient [user](https://github.com/archimede?tab=overview&from=2024-03-01&to=2024-03-23), which is only contribution appears to be the account creation on 28 febbruary 2016.
The name is very important for the project, also for our heritage as Sicilians (peaple born in Sicily? Don't know how to call us in english words).

## Problems

We saw that other people had the same issue, each one with mixed result. It appear to be a common problem, and not an a isolated one.
Just to cite a few discussion reguarding it: [meshcentral organizzation](https://github.com/orgs/community/discussions/65585), [AntiUtopia](https://github.com/orgs/community/discussions/65449), [Cadwyn](https://github.com/orgs/community/discussions/113465), [abandoned user name](https://github.com/orgs/community/discussions/22643).
We can find so much more cases just quering "[name squatting](https://github.com/orgs/community/discussions?discussions_q=name+squatting+)" in discussion

## Discrepancy in the policies

In the GitHub username policy, under "[Name squatting](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy#name-squatting-policy)" ,it stated quite clearly that "account names may not be reserved or inactively held for future use".
Unfortunatly GitHub prevents us to open support requests about this problem.
![image](https://github.com/community/community/assets/73197599/36bc9c69-ab39-4211-99d6-15a43ea39659)

## Final request

In conclusion I would like to ask for help in order to use the name "archimede" for our new organizzation, which it will also be used to create public repository, in order to share with other people future project and ideas. 
Thank you for any help or time spent reading this.

## Post scriptum

The topic area I firmly believe is correct. It is a bug the policies indicates something you can not do, but people do it anyway, breaking the above mentioned rule (name reservation)

**Comment by ebndev** [comment]
Closing and locking this discussion as it is a duplicate and answered [here](https://github.com/orgs/community/discussions/132645#discussioncomment-10086676)
